src/rdf4cpp/rdf/parser/RDFFileParser.hpp

```diff
@@ -0,0 +1,68 @@
+#ifndef RDF4CPP_RDFFILEPARSER_HPP
+#define RDF4CPP_RDFFILEPARSER_HPP
+
+#include "IStreamQuadIterator.hpp"
```

Comment:
For the sake of consistency with the rest of rdf4cpp I would include this as 
`#include <rdf4cpp/rdf/parser/IStreamQuadIterator.hpp>`

This of course works, its just very minor a consistency thingy